### PR TITLE
Added API1600 support for hyp mgr and svr cert

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added API1600 support for hyp mgr and svr cert

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Oneview Features supported
 - Enclosure
 - Enclosure Group
-- Interconnects
+- Hypervisor Manager
+- Interconnect
```
Comment (chebroluharika) [inline CHANGELOG.md:13]: Please remove this as it is not related to our change

Comment (VenkateshRavula) [reply CHANGELOG.md:13]: Since we are mentioning all the resource names in singular form, I made this change.

Comment (chebroluharika) [reply CHANGELOG.md:13]: okay
